components/Document/SubmissionDetails.tsx

```diff
@@ -133,6 +174,22 @@ const styles = StyleSheet.create({
   dot: {
     color: colors.MEDIUM_GREY(),
   },
+  dotWithMargin: {
+    marginLeft: 5,
+  },
+  rscIcon: {
+    verticalAlign: "text-top",
+    marginLeft: 5,
+  },
+  rscText: {
+    fontWeight: 600,
+    color: colors.ORANGE_DARK2(),
```

Comment:
I'm not a fan of these empty function calls for hard string literals. It's fine for now. but let's just avoid these all together.